content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Using OpenTelemetry Instrumentation Libraries with Datadog SDKs
+further_reading:
+    - link: 'tracing/guide/instrument_custom_method'
+      text: 'Instrument a custom method to get deep visibility into your business logic'
+    - link: 'tracing/connect_logs_and_traces'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/visualization/'
+      text: 'Explore your services, resources, and traces'
+    - link: 'https://www.datadoghq.com/blog/opentelemetry-instrumentation/'
+      text: 'Learn More about Datadog and the OpenTelemetry initiative'
+algolia:
+  tags: ['otel instrumentation']
+---
+
+[Instrumentation](/tracing/trace_collection/) is the process of adding code to your application to capture and report observability data.
+[Automatic instrumentation](/tracing/trace_collection/automatic_instrumentation/) is a way to instrument applications and libraries without directly modifying their source code.
+Both OpenTelemetry and Datadog provide automatic instrumentations as part of their SDKs.
+
+Datadog SDKs support adding [instrumentation libraries](https://opentelemetry.io/docs/concepts/instrumentation/libraries/) from OpenTelemetry to their existing automatic instrumentations.
+This provides observability for libraries not originally covered by Datadog SDKs without needing to change SDKs.
+
+## Prerequisites
+
+Before adding OpenTelemetry instrumentation libraries set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
```

Comment:
We may want to restructure where this statement lives because its not a prereq for all languages (e.g, Go)

Reply:
@mtoffl01 How would you suggest restructuring this?

This was called out [here](https://github.com/DataDog/documentation/pull/23819#discussion_r1672500748), but we couldn't identify a better way to frame the information

Reply:
One option would be to leave it where it is and add a qualifier with the languages that require it (e.g, "If you use Java, X, Y or Z, then you must set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.")
Alternatively we can cut this section and just list it under the language tabs that require this env var - I see the env var is mentioned in the Java specific section under "Getting Started" but I if the point is to highlight it **more**, then we could move it out to its own section within the Language specific instructions, like before "compatibility requirements" or something.

Reply:
Also: IMO there is some secondary "harm" in documenting it as a prereq for all languages when it comes to troubleshooting/support. 
Our customers and our support team will use our docs as a source of truth and may incorrectly attribute unexpected behavior to the env var (or lack thereof)

Reply:
The general view was that it was better to have this called out here for visibility, even though it is not strictly necessary for a couple of languages: https://github.com/DataDog/documentation/pull/23819#discussion_r1673680945 (setting it on those languages won't do any harm)

Reply:
Adding qualifiers, etc. makes it more likely customers will miss out this detail or get confused about when to set it.

Either we just remove it and have each language decide whether to call out that you need to set it (i.e. how it was before) or we leave it in place as a blanket setting.

IMHO removing it is more likely to lead to support tickets than leaving it in place...

Reply:
Discussed with Stuart on slack — we decided to keep the section, but add a qualifier that this is not required in Go.

Reply:
d5c6fb8e10